scripts/start_all.sh

```diff
@@ -11,7 +11,12 @@ cdir=`cd $(dirname $0); pwd`
 
 (
     cd $cdir/../deploy
+
+    certsdir=$cdir/../data/certs
+    openssl req -newkey rsa:2048 -nodes -keyout $certsdir/domain.key -x509 -days 365 -out $certsdir/domain.crt -subj "/C=CN/ST=BJ/L=BJ/O=Example/OU=IT/CN=f5-adcaas.com/emailAddress=f5-adcaas@example.com"
```

Comment:
Generate the key pair every time when `start_all.sh`?

Reply:
Could you please change `@example.com` `Example`? we are not an example. 
`f5-adcaas.com`??

Reply:
How about `openssl req -newkey rsa:2048 -nodes -keyout $certsdir/domain.key -x509 -days 365 -out $certsdir/domain.crt -subj "/C=CN/ST=BJ/L=BJ/O=F5Networks"`. 
We only use it as traffic encryption.

Reply:
I think it is an example, users will use there own cert and key in real production env.

Reply:
Generate the key pair every time when start_all.sh?

Reply:
Yes，I really do not what to keep a key file, it is not secure either for test or example env, everyone can download the private key right?

Reply:
No one can download key pairs, I think. 

The problem of generating kp everytime is you may overwrite user's ones if they don't change the path. 
Why not make it re-doable. Like download_*. They can be called any times without any negative effect.